docs/api/1.x/quotas.rst

```diff
@@ -0,0 +1,91 @@
+.. _api-quotas:
+
+Quotas management
+#################
+
+When the built-in plugin ``kinto.plugins.quotas`` is enabled in
+configuration, it becomes possible to configure quotas for a bucket or
+a collection.
+
+Clients can check for the ``quotas`` capability in the
+:ref:`root URL endpoint <api-utilities>`.
+
+.. note::
+
+    In terms of performance, enabling this plugin generates two or
+    three additional queries on backends per request.
+
+* The bucket quota consider the bucket attributes, groups attributes,
+  collections attributes, records attributes.
+* Deleted items size is considered to be null so if you add something
```

Comment:
`Deleted items'`, with the apostrophe to indicate possession (the size of the deleted items). I'd probably reword this as "Deleted items are considered to have zero size" or "size zero".